src/components/form/InviteeForm.js

```diff
@@ -56,22 +71,25 @@ export default function InviteeForm(props: PropsType) {
             />
           </Grid>
           <Grid item>
-            {generatePartnersTextField('name', 'Your partner\'s full name')}
+            {generatePartnersTextField('name', 'Your full name')}
           </Grid>
           <Grid item>
             {partnerState['isGoogler'] ?
                 generatePartnersTextField('email',
                     'Your partner\'s @google address') :
-                <InviteeFormDatePicker
-                    label={'When is your partner going to join Google?'}
-                    propagateNewData={changePartnerState('startDate')}/>}
+                <div>
+                  {generatePartnersTextField('email', 'Your partner\'s @noogler.google.com address')}
```

Comment:
Before joining Google, the Noogler don't have any ...google.com address, not even a @noogler.google.com one.